docker/Dockerfile

```diff
@@ -1,13 +1,18 @@
-ARG SPARK_VERSION=2.3.1-2.2.1-2-hadoop-2.6
+ARG SPARK_VERSION=2.7.0-2.4.0-hadoop-2.7
 FROM mesosphere/spark:$SPARK_VERSION
 
-ARG SPARK_VERSION
```

Comment:
SPARK_VERSION needs to be defined (again) after the FROM, otherwise it will not be available when defining `ENV SPARK_MESOS_EXECUTOR_DOCKER_IMAGE mesosphere/spark:$SPARK_VERSION`

Reply:
ok - you are right. I will fix that

Reply:
added